docs/router-tutorial/react-navigation-stack/react-navigation-stack.md

```diff
@@ -31,7 +31,7 @@ In this chapter, we'll try installing the react-navigation-stack, and we'll give
 - react-native-cli : 2.0.1
 - react-navigation: 4.0.10
 
-### RN install & **RN Navigation stack install**
+### React-native install & **React-native Navigation stack install**
```

Comment:
`React Native install & React Navigation Stack install`

Reply:
Thx. i remove dash and use uppercase now